cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-db-operations.asciidoc

```diff
@@ -1,44 +0,0 @@
-The rules of a security model may impact some of the database operations.
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/limitations.adoc